core/src/main/java/nl/nn/adapterframework/configuration/digester/DigesterRule.java

```diff
@@ -38,6 +38,11 @@
 	 */
 	private @Getter @Setter String selfRegisterMethod;
 
+	/**
+	 * The 'callMethod()' attribute.
```

Comment:
Kan je (hier) uitleggen wat het verschil is tussen 'callMethod' en 'registerMethod'?